src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -14994,8 +14940,6 @@ void VRFCondenserEquipment::VRFOU_PipeLossH(
     using General::SolveRoot;
 
     int TUListNum;      // index to TU List
-    int TUIndex;        // Index to terminal unit
-    int CoilIndex;      // index to coil in terminal unit
```

Comment:
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:14997]:(style),[variableScope],The scope of the variable 'TUIndex' can be reduced.
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:14998]:(style),[variableScope],The scope of the variable 'CoilIndex' can be reduced.